linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_mac_modification_usr_share/oval/shared.xml

```diff
@@ -4,6 +4,14 @@
       mandatory access controls (SELinux) in usr/share/selinux are enabled.") }}}
 
     <criteria operator="OR">
+{{%- if 'sle15' in product -%}}
+      <criteria operator="AND">
+        <criterion test_ref="check_selinux_policy_devel_installed"
+          comment="Check no selinux-policy-devel package installed" negate="true"/>
+        <criterion test_ref="check_selinux_policy_targeted_installed"
+          comment="Check no selinux-policy-targeted package installed" negate="true"/>
+      </criteria>
+{{%- endif -%}}
```

Comment:
Have you considered creating a platform restriction in the rule.yml instad of modifying the OVAL? What are the reasons for this solution?

Reply:
The main reason here is that for sle platform, not sure about the others, we need to have some of the selinux-policy packages installed in order the whole rule to be relevant because otherwise even if we have selinux installed with no policies there is nothing to audit

Reply:
That doesn't explain why you chose this way, you can get the same result with using platform expressions. Also if you used platforms you would get "notapplicable" results if the package isn't present, and that way you can get more understandable user experience in the HTML report.

Reply:
So I assume your suggestion to rework it, so the rule is limited by platform clause.
platform: package[selinux-policy-devel] or [selinux-policy-targeted] ?
and add declaration of the relevant packages for all linux flavours?